merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/InsertActivityTypesAction.java

```diff
@@ -0,0 +1,71 @@
+package gov.nasa.jpl.aerie.merlin.server.remotes.postgres;
+
+import gov.nasa.jpl.aerie.merlin.server.models.ActivityType;
+import org.intellij.lang.annotations.Language;
+
+import java.sql.BatchUpdateException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Collection;
+
+import static gov.nasa.jpl.aerie.merlin.driver.json.ValueSchemaJsonParser.valueSchemaP;
+
+/*package-local*/ final class InsertActivityTypesAction implements AutoCloseable {
```

Comment:
Huge fan of the `Create -> Insert` rename as well as the batching functionality :100: